### PR TITLE
Add branch name while importing YAML in azdo

### PR DESCRIPTION
**Pull request by maartenvandiemen**
**Add branch name while importing YAML in azdo**

## Checklist 
Mark completed with "x" between brackets, "[x]", or checking the box once the PR is created:
- [ ] Has related GitHub Issue 💥 [Create Issue](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#reporting-issues) 📝
- [X] Tested it
- [x] Read the PR collaboration guide 👓 [Collaboration Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#pull-requests) 📝

Changes proposed in this pull request:

- Add name of the main branch explicitly while importing YAML pipeline during labs
- Fix style issues (AZ400_M01_L01)

### CODE REVIEW

File: Instructions/Labs/AZ400_M05_L10_Integrating_Azure_Key_Vault_with_Azure_DevOps.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ lab:
 - This lab requires **Microsoft Edge** or an [Azure DevOps supported browser.](https://learn.microsoft.com/azure/devops/server/compatibility)
 
 - **Set up an Azure DevOps organization:** If you don't already have an Azure DevOps organization that you can use for this lab, create one by following the instructions available at [Create an organization or project collection](https://learn.microsoft.com/azure/devops/organizations/accounts/create-organization).
-
+**
```
Comment (yashints) [inline Instructions/Labs/AZ400_M05_L10_Integrating_Azure_Key_Vault_with_Azure_DevOps.md:16]: Is this intentional?

File: Instructions/Labs/AZ400_M01_L01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md [hunk 2]
```diff
@@ -232,15 +232,15 @@ The sprint backlog should contain all the information the team needs to successf
 
 1. Select the **Capacity** tab of the **Sprints** view.
 
-![Sprint Capacity View](images/m1/EShop-WEB-capacity_v1.png)
+    ![Sprint Capacity View](images/m1/EShop-WEB-capacity_v1.png)
```
Comment (yashints) [inline Instructions/Labs/AZ400_M01_L01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md:235]: revert back pls

File: Instructions/Labs/AZ400_M01_L01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md [hunk 3]
```diff
@@ -232,15 +232,15 @@ The sprint backlog should contain all the information the team needs to successf
...
 
     > **Note**: This view enables you to define what activities a user can take on and at what level of capacity.
 
 1. On the **Capacity** tab of the **Sprints** view, click **+Add User** and select your user account. For this user, set the **Activity** field to **Development** and, in the **Capacity per day** textbox, type **1**.
 
     > **Note**: This represents 1 hour of development work per day. Note that you can add additional activities per user in the case they do more than just development.
 
-![Set Development Capacity for a user](images/m1/EShop-WEB-capacity-setdevelopment_v1.png)
+    ![Set Development Capacity for a user](images/m1/EShop-WEB-capacity-setdevelopment_v1.png)
```
Comment (yashints) [inline Instructions/Labs/AZ400_M01_L01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md:243]: revert back pls

File: Instructions/Labs/AZ400_M01_L01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md (hunk 2 above)
Comment (maartenvandiemen) [reply Instructions/Labs/AZ400_M01_L01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md:235]: Reason why I did this is due to the following issue how it's rendered:
image

While I would expect something like this:
image

File: Instructions/Labs/AZ400_M01_L01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md (hunk 3 above)
Comment (maartenvandiemen) [reply Instructions/Labs/AZ400_M01_L01_Agile_Planning_and_Portfolio_Management_with_Azure_Boards.md:243]: Reason why I did this is due to the following issue how it's rendered:
image

While I would expect something like this:
image

**Pull request merged**